cloudflare/src/backends.js

```diff
@@ -1,5 +1,5 @@
 
-const {isStaticUrl, isAlertsUrl, isDrupalResponse, isMediaDownloadUrl, isValidRedirect, isFileRedirect} = require('./util');
```

Comment:
Did you mean to remove isDrupalResponse?

Reply:
Yep, it was unused.